buildchow/readme.adoc

```diff
@@ -0,0 +1,75 @@
+BuildChow
+=========
+
+BuildChow is the nickname for templates of JBoss Tools and Developer
+Studio jenkins jobs.
+
+By using templates we do not have to rely on jenkins UI to manually update jobs
+or bulk-update in large set of config.xml files.
+
+BuildChow uses http://ci.openstack.org/jenkins-job-builder[Jenkins Job
+Builder](jjb) from Openstack.org.
+
+== Requirements
+
+. Python 2.7+
+. Python PIP
+. Git
+
+On Fedora you get this by runing: `yum install -y python-pip git`
+
+== How to use
+
+First need to install jjb.
```

Comment:
Can we rename "BuildChow" to "Jenkins Job Builder" instead? I've never understood the use of "chow" or "build" in the context of "templatizing Jenkins jobs".